tests/integration/Social/Services/CommentsCest.php

```diff
@@ -113,4 +115,17 @@ public function commentReaction(IntegrationTester $I): void
         $reaction = Reactions::addMessageReaction('☺', new Users(), $this->comment);
         $I->assertEquals($this->comment->getId(), $reaction->entity_id);
     }
+
+    /**
+     * Test Users Comments Interaction
+     *
+     * @param IntegrationTester $I
+     * @before getCommentData
+     * @return void
+     */
+    public function messageInteraction(IntegrationTester $I): void
+    {
+        $messageInteraction = Interactions::add(new Users(), $this->comment, ModelsInteractions::REACT);
```

Comment:
@arfenis one comment why not just call the metho directly in the $I->assertFalse ?

Reply:
Forgot, fixing.